lib/actionview_precompiler/ast_parser/ripper.rb

```diff
@@ -34,7 +34,11 @@ def fcall_named?(name)
       def argument_nodes
         raise unless fcall?
         return [] if self[1].nil?
-        self[1][0...-1]
+        if self[1].last.is_a?(FalseClass) || self[1].last.type == :vcall
```

Comment:
I was trying to think of other things that could be here that we would want to ignore. Like potentially there are other kinds of proc/function AST nodes that we should be aware of, but I wasn't able to find and test any others.